include/CLI/StringTools.hpp

```diff
@@ -45,7 +45,7 @@ using enums::operator<<;
 namespace detail {
 /// a constant defining an expected max vector size defined to be a big number that could be multiplied by 4 and not
 /// produce overflow for some expected uses
-constexpr int expected_max_vector_size{1 << 29};
+CLI11_MODULE_INLINE constexpr int expected_max_vector_size{1 << 29};
```

Comment:
By the way, can we not have `inline` here all the time?

Reply:
seems to work if I condition on C++17 and have it be inline or static,  will go with that.